docs/sphinx/user_guide/feature/reduction.rst

```diff
@@ -128,8 +128,8 @@ There are some important constraints note about RAJA reduction usage.
 
 * ``cuda_reduce`` - Reduction policy for use with CUDA execution policies that uses CUDA device synchronization when finalizing reduction value.
 
-* ``cuda_reduce_async`` - Reduction policy for use with CUDA execution policies that may not use CUDA device synchronization when finalizing reduction value.
+.. ## * ``cuda_reduce_async`` - Reduction policy for use with CUDA execution policies that may not use CUDA device synchronization when finalizing reduction value.
```

Comment:
Please make the list syntax the same for all items.  Also, is the word 'may' correct here; i.e., when using an async policy device synchronization may occur during finalization? If this is correct, please note the conditions that determine whether or not synchronization will happen.

Reply:
This bit of documentation may not be necessary without the async template argument

Reply:
Unless I've misunderstood, the ".." syntax indicates a comment in the .rst file. I was attempting to comment the line out, and I _think_ that is correct as is.

Reply:
I see. In that case, you should remove the comment because it is the same as the list item above it. Same for the one below.